### PR TITLE
Temoa Model PR -- CLEAN

### LINKED ISSUE #28: Generate a Temoa database for the UIUC grid system
This issue can be closed when a SQL database that represents the UIUC embedded grid has been created.

### PR DESCRIPTION
**Pull request by samgdotson**
This PR does not include anything from the presentation branch. 
Closes #28 by generating a TEMOA model of the UIUC energy grid in a SQL database.

Future work will be to add a config file to run from the command line rather than the temoa cloud interface.

### CODE REVIEW

**Comment by samgdotson** [comment]
Delete this branch after merging.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,19 @@ Multiple scenarios will be addressed:
 3. Increasing penetration of variable renewable energy sources.
 4. Add grid flexibility in the form of H2 and thermal storage.
 
+## Instructions to Run TEMOA
+TEMOA is an open source modeling tool available on GitHub (link). Follow the installation instructions here (link).
```
Comment (katyhuff) [inline README.md]: links

File: README.md [hunk 2]
```diff
@@ -8,6 +8,19 @@ Multiple scenarios will be addressed:
 3. Increasing penetration of variable renewable energy sources.
 4. Add grid flexibility in the form of H2 and thermal storage.
 
+## Instructions to Run TEMOA
+TEMOA is an open source modeling tool available on GitHub (link). Follow the installation instructions here (link). 
+
+After creating a database in sql, navigate to the directory with your database:
```
Comment (katyhuff) [inline README.md:14]: I'm surprised they don't have a utility for generating this SQL database with a simpler input file syntax and use of sqlite3. Nothing to change here... just a comment.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,19 @@ Multiple scenarios will be addressed:
...
+
+``sqlite3 [filename].sqlite < [filename].sql``
+
+if you don't have sqlite installed, run:
+
+``sudo apt-get install sqlite``
```
Comment (katyhuff) [inline README.md]: not `sudo apt-get install sqlite3` ?

**Review by katyhuff — COMMENTED** [review]
I didn't go through the whole PR but I had time for a couple of comments. Will do my best to get back to it, but if I don't, please go on without me.

File: README.md (hunk 3 above)
Comment (samgdotson) [reply README.md]: I did ``sqlite`` and run ``sqlite3`` just fine. I'm not sure if there is a difference or it's sort of like specifying ``python`` vs ``python3`` (pre-2020, that is).

**Review by robfairh — APPROVED** [review]
LGTM

**Pull request merged**